docker/x86_64-musl/Dockerfile

```diff
@@ -1,12 +1,19 @@
-FROM debian:stretch
+FROM alpine:edge
```

Comment:
Is it possible to remain on debian?

Reply:
I have found no working configuration with `musl-gcc` on `debian`.